src/Ng/CommandHelpers.cs

```diff
@@ -363,5 +374,52 @@ public static Func<HttpMessageHandler> GetHttpMessageHandlerFactory(bool verbose
 
             return handlerFunc;
         }
+
+        public static IEnumerable<EndpointFactory.Input> GetEndpointFactoryInputs(IDictionary<string, string> arguments)
+        {
+            return arguments.GetOrThrow<string>(Arguments.EndpointsToTest).Split(';').Select(e => {
```

Comment:
Please add a comment on what the endpoints are supposed to look like, some examples.

Reply:
There are some in `Arguments.EndpointsToTest`.